tox.ini

```diff
@@ -151,6 +148,25 @@ deps=https://github.com/twisted/pydoctor/archive/3f9c64829dfa040b334c9ae27c332c7
 [testenv:manifest-checker]
 skip_install = true
 
+[testenv:lint]
+skip_install=True
+basepython=python3.6
+
+[testenv:black]
+skip_install=True
+basepython=python3.6
```

Comment:
In this PR, the basepython for running black is Python 3.6.
However, Python 3.5 support has not yet been dropped from Twisted.

Do you need to specify `--target` to black, to prevent style changes incompatible with Python 3.5 from going in?

I'm not in favor of pinning the tox environment for black to a specific Python version,
because a Twisted contributor may have a different version of Python installed on their system,
and `tox -e black` will not work for them.

Reply:
I believe that the `target-version` in `pyproject.toml` ensures that 3.5-compatible syntax is used.

Reply:
I think you are correct.  That's good enough then.